docs/data_preparation.rst

```diff
@@ -641,20 +641,31 @@ Other Data (multivec)
 
 Multivec files are datatype agnostic. For use with generic data, create a
 `segments` file containing the maximum value for each segment. A segment is an
-arbitrary set of discontinuous blocks that the data is partitioned into. If the
+arbitrary set of discontinuous blocks that the data is partitioned into. In the
+case of genomics data, segments correspond to chromosomes. If the
 data has no natural grouping, one segment can be used which contains the
 maximum x value in the dataset:
```

Comment:
> one segment can be used which contains the maximum x value in the dataset

I am not sure I get what this means. What does the 20000 and 40000 in the list below refer to? Maybe I am just missing context but I am not sure what the segment has to do with some maximum value.

Reply:
I updated the docs. This was originally designed with chromosomes in mind. So the 20000 and 40000 are the chromsizes. They're the lengths that segments will occupy along the x-axis. So if you have two segments:

segment1 20000
segment2 40000

And a datapoint at `segment 2 0`, it will actually get drawn at position 20000 on the x-axis because the first 20000 is occupied by the first segment. Does that make sense?

Reply:
Oh so `segment1: 10 -100 [5, 15, 20]` would be a data point in `segment1` that spans `[10-100]` and has 3 values attached to it? Just to be sure I get the idea: would this datapoint be ultimately be merged with other data points of segment1 or would it be visualized irrespectively of the segment it's lying in? I guess my question is what exactly is the segment definition good for in terms of the visual representation?